### PR TITLE
Feature/cleanup requirements

### PR DESCRIPTION
**Pull request by dsmits**
Moved requirements that are only necessary for tests, checking and build to github actions script.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,8 +1,3 @@
 vantage6
```
Comment (svenvanderburg) [inline requirements.txt:1]: If you're editing stuff here anyway, you might as well add versions for your requirements.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -29,7 +29,9 @@ jobs:
           pip install -r requirements.txt
       - name: Check style against standards using prospector
         shell: bash -l {0}
-        run: prospector -o grouped -o pylint:pylint-report.txt
+        run: |
+            pip install prospector[with_pyroma]
```
Comment (svenvanderburg) [inline .github/workflows/build.yml:33]: Perhaps you could add a `requirements-test.txt` where you specify you dependencies for testing. Here you can then also fix versions.

**Review by svenvanderburg — APPROVED** [review]
I've added some comments, but feel free to ignore!

File: requirements.txt (hunk 1 above)
Comment (dsmits) [reply requirements.txt:1]: I'd rather not prematurely fix the versions, only if it gives a problem. I like to have the latest versions of everything if possible.

File: .github/workflows/build.yml (hunk 2 above)
Comment (dsmits) [reply .github/workflows/build.yml:33]: created requirements-test

**Comment by dsmits** [comment]
I will try creating  a new PR to see if that reflects my latest changes

**Pull request closed without merging**